Sources/Logging/Strings/NetworkStrings.swift

```diff
@@ -86,7 +87,10 @@ extension NetworkStrings: CustomStringConvertible {
             return "We can't find the cached response, but call has already been retried. " +
                 "Returning result from backend \(response)"
 
-        case .blocked_network(let url, let newHost):
+        case let .failing_url_resolved_to_host(url, resolvedHost):
+            return "Failing url '\(url)' resolved to host '\(resolvedHost)'"
```

Comment:
Would we have any secret info in the URL? I would guess not but just checking we don't leak any info in the logs.

Reply:
I don't think so, all that data is normally in request bodies.